City-of-Helsinki/drupal-module-helfi-tpr
# UHF-3505: Migration fixes

To test this:
- `composer require drupal/helfi_tpr:dev-UHF-3505 -W`
- `drush cr`

Run all migrates:

- `MIGRATE_LIMIT=100 drush mim tpr_unit`
- `drush mim tpr_service`
- `drush mim tpr_errand_service`
- `drush mim tpr_service_channel`

All migrations (besides tpr_unit) should be really fast now. 

Relevant api-base change: https://github.com/City-of-Helsinki/drupal-module-helfi-api-base/pull/41
